.github/workflows/e2e_targeted.yaml

```
@@ -88,6 +88,21 @@ jobs:
       run-istio: ${{ inputs.run-istio }}
       all-supported-k8s-versions: ${{ inputs.all-supported-k8s-versions }}
 
+  run-unreleased-kong:
+    needs: choose-image
+    if: ${{ !cancelled() }}
+    uses: ./.github/workflows/_e2e_tests.yaml
+    secrets: inherit
+    with:
+      kic-image: kong/nightly-ingress-controller:nightly
```

```suggestion
      kic-image: ${{ needs.choose-image.outputs.image }}
      load-local-image: ${{ inputs.controller-image == '' }}
```
Just to be consistent with this workflow's inputs. `needs.choose-image.outputs.image` will contain a locally built image (if `inputs.controller-image` is empty) or the one that's passed in the `inputs.controller-image`.